app/src/main/java/com/conupods/MapsActivity.java

```diff
@@ -11,10 +13,15 @@
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.MarkerOptions;
 
-public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+public class MapsActivity extends FragmentActivity implements OnMapReadyCallback, View.OnClickListener {
 
     private GoogleMap mMap;
 
+    private LatLng sgw = new LatLng(45.496080, -73.577957);
+    private LatLng loy = new LatLng(45.458333, -73.640450);
```

Comment:
These are technically constants right? Should declare them as final. We could declare them in an xml (the numbers at least) file and refer to them by uppercase letters in case we need to use them somewhere else because I doubt this class will remain like this.

Reply:
+1

Reply:
done